docs/index.html

```diff
@@ -1,7 +1,7 @@
 <html>
 
 <head>
-  <script src="https://s3.us-east-2.amazonaws.com/oyster-webnode/static/js/oyster-webnode.js" eth-address="TODO" />
+  <script src="https://s3.us-east-2.amazonaws.com/oyster-webnode/static/js/oyster-webnode.js" eth-address="TODO"></script>
```

Comment:
@EdmundMai is there a way to point this to whatever is on master in this repo? so our e2e tests are always testing the latest code?

Reply:
@AaronVasquez Want me to go ahead and approve this or did you want to change something here?

Reply:
approve please. I can create a new PR for more changes